lib/cli.js

```diff
@@ -172,7 +172,12 @@ function runDevelopmentMode (testModulePath) {
         console.log(chalk.red('Error: step number is not exists in test'))
         return
       }
-      yield runTestWhileConditionMet(() => !(tester.getCurrentStepNumber() === String(args.stepNumber)))
+      const matchCondition = () => {
+        const currentStep = tester.getCurrentStepNumber()
+        const targetStep = String(args.stepNumber)
+        return (currentStep === targetStep || currentStep === `${targetStep}.1`)
```

Comment:
There is a case where e.g. `7.1.1.1` might not be matched when you say `runto 7`.

To fix this, let’s match by prefix:

```js
return `${currentStep}.`.startsWith(`${targetStep}.`)
```